src/Nethermind/Nethermind.Blockchain/Synchronization/SyncConfig.cs

```diff
@@ -42,5 +42,6 @@ public class SyncConfig : ISyncConfig
         public int BeamSyncPreProcessorTimeout { get; set; } = 15;
         public bool BeamSyncFixMode { get; set; } = false;
         public bool BeamSyncVerifiedMode { get; set; } = false;
+        public long[] TotalDifficultyOverrides { get; set; }
```

Comment:
can we add it in Chainspec instead?

Reply:
We can, the question is do we want to introduce more incompatibilities with parity chainspec? What do you think?